packages/vite/import-meta.d.ts

```diff
@@ -0,0 +1,10 @@
+import type {
+  ImportMeta as ViteImportMeta,
+  ImportMetaEnv as ViteImportMetaEnv
+  // eslint-disable-next-line node/no-missing-import -- use .js for `moduleResolution: "nodenext"`
```

Comment:
When `vite/client` or `vite/import-meta` is used from a file using `moduleResolution: "nodenext"`, this import needs to be `vite/client/types` or `vite/client/types.js`... (vitepress uses `moduleResolution: "nodenext"`)